models/request/draft_transaction_json.go

```diff
@@ -36,6 +43,14 @@ func expandOutputForMarshaling(output Output) (any, error) {
 			Type:   o.GetType(),
 			Output: &o,
 		}, nil
+	case paymailreq.Output:
```

Comment:
Will you consider my suggestion about this part of code I gave you some time ago?
- Or you're fine with this solution and "if it works - don't touch it?"